pkg/antctl/raw/supportbundle/command.go

```diff
@@ -120,21 +120,18 @@ func init() {
 	}
 }
 
-var getSupportBundleClient func(cmd *cobra.Command) (systemclientset.SupportBundleInterface, error) = setupSupportBundleClient
+var getSupportBundleClient func() (systemclientset.SupportBundleInterface, error) = setupSupportBundleClient
 
-func setupSupportBundleClient(cmd *cobra.Command) (systemclientset.SupportBundleInterface, error) {
-	kubeconfig, err := raw.ResolveKubeconfig(cmd)
-	if err != nil {
-		return nil, err
-	}
+func setupSupportBundleClient() (systemclientset.SupportBundleInterface, error) {
+	kubeconfig := &rest.Config{}
```

Comment:
I didn't see any reason to import kubeconfig for SetupLocalKubeconfig instead of creating a new one (I tried it works well with the newly created kubeconfig). Could you check if there was any reason for this in the previous design? @antoninbas @wenyingd

Reply:
I agree with you. We override all the important fields in `SetupLocalKubeconfig`, so I am not sure why it was written this way to begin with.